css/index.css

```diff
@@ -140,39 +174,97 @@ p,
 ul {
   font-size: 1.6rem;
 }
+@media (max-width: 500px) {
+  h2,
+  h3,
+  p,
+  ul {
+    text-align: center;
+    padding: 5px;
+  }
+}
 nav {
   display: flex;
   justify-content: space-around;
-  margin-top: 1%;
+  margin: 5%;
+  padding: 5px;
+}
+@media (max-width: 500px) {
+  nav {
+    flex-direction: column;
+    margin: 5px;
+  }
+}
+@media (min-width: 1000px) {
+  nav {
+    width: 1000px;
+  }
+}
+a:hover {
+  color: forestgreen;
+}
+@media (max-width: 500px) {
+  a {
+    border: 1px solid black;
+    padding: 20px;
+    text-align: center;
+  }
 }
 .container {
   Margin: 5%;
   display: flex;
   justify-content: center;
   flex-direction: column;
-  background-color: white;
+  background-color: #877EC1;
+  animation: colorChange 60s ease;
+}
+@media (min-width: 1000px) {
+  .container {
+    width: 1000px;
+  }
+}
+@media (max-width: 500px), (max-width: 800px) {
+  .container {
+    margin: 20px;
+    animation: none;
+  }
+}
+@media (max-width: 500px) and (max-width: 500px), (max-width: 800px) and (max-width: 500px) {
+  .container {
+    background-color: white;
+  }
 }
 .topContainer {
   display: flex;
   flex-direction: column;
 }
 .topContainer img {
-  width: 50%;
-  align-self: center;
+  width: 100%;
+}
+@media (max-width: 500px) {
```

Comment:
nest media query under img. After doing all the nesting,  it should look something like this: 

```
.container{
	margin: auto;
	display:flex;
	justify-content: center;
	flex-direction: column;
	background-color:#877EC1;
	animation:colorChange 60s ease;
	
	@media @large{
		.maxWidth();
	}
	
	@media @mobile, @tablet{
		margin: 20px;
		animation: none;
		@media @mobile{
			background-color:white;
		}
	}

	.topContainer{
		display:flex;
		flex-direction: column;
		background-color: white;
		
		img{
			width: 100%;
			@media @mobile{
				width: 100%;
			}
		}
	}
```

Please have everything nested, including the <a> elements in the <nav>.